doc/cephfs/createfs.rst

```diff
@@ -48,7 +48,36 @@ Once the pools are created, you may enable the file system using the ``fs new``
 
 .. code:: bash
 
-    $ ceph fs new <fs_name> <metadata> <data>
+    $ ceph fs new <fs_name> <metadata> <data> [--force] [--allow-dangerous-metadata-overlay] [<fscid:int>] [--recover]
+
+This command creates a new file system. The file system name and metadata pool
+name are self-explanatory. The specified data pool is the default data pool and
+cannot be changed once set. Each file system has its own set of MDS daemons
+assigned to ranks so ensure that you have sufficient standby daemons available
+to accommodate the new file system.
+
+The ``--force`` option is used to achieve any of the following:
+
+- To set an erasure-coded pool for the default data pool.
+  Use of an EC pool for the default data pool is discouraged.
```

Comment:
Maybe include a brief note as to why this is discouraged?  The reasons given here?

https://documentation.suse.com/ses/6/html/ses-all/cha-ceph-as-cephfs.html#ceph-cephfs-cephfs-create

which seem like the usual EC vs replication tradeoffs vs anything CephFS specific?

Reply:
Guidelines about creating pools for file system is available just before this section.
https://docs.ceph.com/en/latest/cephfs/createfs/#creating-pools